### ISSUE TITLE
vmalert-tool-prod does not support non-integer series test values

### ISSUE THREAD

**Issue by shric** [VictoriaMetrics/VictoriaMetrics#6391]
### Describe the bug

I ran:

`$ ./vmalert-tool-prod unittest --disableAlertgroupLabel --files foo.test.yml`

I expect:
```
Unit Testing:  foo.test.yml
evaluation_interval set to 1m by default
  SUCCESS
2024/06/02 09:23:32 Total time: 3.171022603s
```
I got:
```
Unit Testing:  foo.test.yml
evaluation_interval set to 1m by default
  FAILED

failed to run unit test for file "foo.test.yml":
[failed to parse input series value 1 1 1 0.1 0.1 0.1 0.1: got invalid operation 101110]2024/06/02 09:20:47 unittest failed
```

### To Reproduce

Run `$ ./vmalert-tool-prod unittest --disableAlertgroupLabel --files foo.test.yml` using: 

`foo.test.yml`:

```
rule_files:
  - foo.yml

tests:
  - interval: 1m
    input_series:
      - series: 'foo{}'
        values: '1 1 1 0.1 0.1 0.1 0.1'

    alert_rule_test:
      - eval_time: 5m
        alertname: "Foo"
        exp_alerts:
          - exp_annotations:
              description: "foo is less than one!"
```

`foo.yml`:

```
groups:
- name: foo
  rules:
  - alert: Foo
    expr: foo < 1
    for: 1m
    annotations:
      description: "foo is less than one!"
```

### Version

```
$ ./vmalert-tool-prod --version
vmalert-tool version vmalert-tool-20240425-145513-tags-v1.101.0-0-g5334f0c2c
2024/06/02 09:27:17 Total time: 38.433µs
```

### Additional information

I thought of this as "feature request", but decided on "bug" for two reasons:

1. Time series data are float64, so why only integers in tests?
2. promtool supports this:
```
$ prometheus-2.52.0.linux-amd64/promtool test rules foo.test.yml
Unit Testing:  foo.test.yml
  SUCCESS
```

If this is accepted as a valid bug, I am happy to submit a PR if you like. I'm also happy for someone else to do it. Let me know.

**Comment by f41gh7** [comment]
@Haleygo  PTAL

**Comment by hagen1778** [comment]
@shric the issue should be addressed in https://github.com/VictoriaMetrics/VictoriaMetrics/commit/e3e40cb8484e0b28af784d7fe38584e4a7a6b23a and will be included to the next release. Meanwhile, vmalert-tool can be build from sources via command `make vmalert-tool` from the root directory of https://github.com/victoriaMetrics/victoriaMetrics repo.

**Comment by shric** [comment]
@hagen1778 @Haleygo that's amazing. So fast, thank you!

I'm not sure if the etiquette is I close the issue, so I will let you close it when you're ready. Thanks again.

**Comment by hagen1778** [comment]
> I'm not sure if the etiquette is I close the issue, so I will let you close it when you're ready. Thanks again.

Worry not! We close bug reports and feature requests only after the corresponding release.

**Comment by valyala** [comment]
The issue should be fixed starting from [v1.102.0-rc1 release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.102.0-rc1).
